File: Java/src/Practica1,2/Circulo.java
package Practica1;

public class Circulo {
	private
	Punto centro;
	float radio;
	
	public Circulo(Punto centro, float radio) {
		this.centro = centro;
		this.radio = radio;
	}
	

	@Override
	public String toString() {
		return "Circulo [centro=" + centro + ", radio=" + radio + "]";
	}
	
	public void dibujaCirculo(){
		System.out.println("Dibujando un circulo en" + centro + "con radio" + radio);
	}


	public static void main(String[] args) {
		Punto centro = new Punto(2.0f, 3.0f);
		Circulo circulo = new Circulo(centro, 5.0f);
		System.out.println(circulo);
		circulo.dibujaCirculo();

	}

}

File: Python/Circulo.py
import math
import Punto

class Circulo:
    def __init__(self, centro: Punto, radio: float):
        self.centro = centro
        self.radio = radio

    def __str__(self):
        return f"Círculo con centro en {self.centro.coord_cartesianas()} y radio {self.radio}"
    def dibuja_circulo(self):
        print(f"Dibujando un círculo en {self.centro.coord_cartesianas()} con radio {self.radio}")

centro = Punto(2.0, 3.0)
circulo = Circulo(centro, 5.0)
print(circulo)
circulo.dibuja_circulo()